tests/unit/badge/badge.renderBadge.spec.js

```diff
@@ -0,0 +1,42 @@
+import { shallow } from 'enzyme';
```

Comment:
no, but `render` can return just result from `renderBadge` in some cases, so I test them separately